addons/website_sale/views/templates.xml

```diff
@@ -1517,14 +1517,33 @@
                     data-name="Alternative Products" style="background-image: none;" t-att-data-filter-id="product._get_alternative_product_filter()"
                     data-template-key="website_sale.dynamic_filter_template_product_product_borderless_1" data-product-category-id="all" data-number-of-elements="4"
                     data-number-of-elements-small-devices="1" data-number-of-records="16" data-carousel-interval="5000" data-bs-original-title="" title="">
-                    <div class="container o_not_editable">
-                        <div class="css_non_editable_mode_hidden">
-                            <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none">
-                                Your Dynamic Snippet will be displayed here...
-                                This message is displayed because youy did not provide both a filter and a template to use.
-                            </div>
+                    <div class="container">
+                        <div class="row s_nb_column_fixed">
+                            <section class="s_dynamic_snippet_title oe_unremovable oe_unmovable d-flex flex-column flex-md-row justify-content-between mb-lg-0 pb-3 pb-md-0">
+                                <div>
+                                    <h4>Alternative Products</h4>
+                                    <p class="lead">check out what else we have for you!</p>
+                                </div>
+                                <div>
+                                    <a
+                                        title="See All"
+                                        class="s_dynamic_snippet_main_page_url d-none"
+                                        href="#">
+                                        See all
+                                        <i class="fa fa-long-arrow-right ms-2"/>
+                                    </a>
+                                </div>
+                            </section>
+                            <section class="s_dynamic_snippet_content oe_unremovable oe_unmovable o_not_editable col">
```

Comment:
when it will be displayed now? :thinking:

Reply:
This message appears when there is neither a filter nor a template for that snippet, which is very rare.
And To display the message, we must explicitly change the values after the class 'missing_option_warning' using js.

Reply:
I am not sure why this section is oe_unremovable and oe_unmovable

Can you check with PO do we really need this section and are they able see it in UX with our needing to change js?